src/Entity/Embeddable/Address.php

```diff
@@ -57,6 +57,11 @@ class Address implements LoggableEntityInterface
      */
     private $bbrData;
 
+    /**
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $extraAddressInformation;
```

Comment:
I think this should be called just `extraInformation` (or `additionalInformation`). We already know that it's related to an address object.

Reply:
Good point. Renamed to `extraInformation`.